### ISSUE TITLE
Create a Project Description

### ISSUE THREAD

**Issue by temanbrcom** [eclipse/che-che4z-lsp-for-cobol#161]
## Description ##
As a software engineer, I would like to have a description of the repository to let people know the goal of the project

## Acceptance Criteria ##
- The repository has a description header 

## Definition of Done ##
- The story is implemented according to the description.

## Technical details ##
To do this, go to ```Insights - Community - Description - add```

**Comment by zeibura** [comment]
Insights - Community - Description - add just loops me back to the home page

**Comment by temanbrcom** [comment]
Yes, probably, we need higher permission to do that. Please, write the description here and we will submit a ticket to add it to the repository.

**Comment by zeibura** [comment]
I think "COBOL Language Support" is enough. I had a look at some descriptions on Zowe repos and they all just name the product.
